src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/PrisonService.kt

```diff
@@ -24,16 +25,24 @@ import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.external.pris
 class PrisonService(
   private val prisonApiClient: PrisonApiClient,
   private val offenceSDSReleaseArrangementLookupService: OffenceSDSReleaseArrangementLookupService,
+  private val featureToggles: FeatureToggles,
   private val botusTusedService: BotusTusedService,
 ) {
   //  The activeDataOnly flag is only used by a test endpoint (1000 calcs test, which is used to test historic data)
-  fun getPrisonApiSourceData(prisonerId: String, activeDataOnly: Boolean = true): PrisonApiSourceData {
+  fun getPrisonApiSourceData(prisonerId: String, supportInactiveSentencesAndAdjustments: Boolean): PrisonApiSourceData {
     val prisonerDetails = getOffenderDetail(prisonerId)
-    val activeOnly = activeDataOnly || prisonerDetails.agencyId != "OUT"
-    return getPrisonApiSourceData(prisonerDetails, activeOnly)
+    return getPrisonApiSourceData(prisonerDetails, supportInactiveSentencesAndAdjustments)
   }
 
-  fun getPrisonApiSourceData(prisonerDetails: PrisonerDetails, activeDataOnly: Boolean = true): PrisonApiSourceData {
+  //  The activeDataOnly flag is only used by a test endpoint (1000 calcs test, which is used to test historic data)
+  fun getPrisonApiSourceData(prisonerId: String): PrisonApiSourceData {
+    val prisonerDetails = getOffenderDetail(prisonerId)
+    val supportInactiveSentencesAndAdjustments = featureToggles?.supportInactiveSentencesAndAdjustments ?: false
```

Comment:
Why `featureToggles?` when the val is defined as non null?

Reply:
Because when we use the integration tests it's leaving the featuretoggles as null - and I didn't want to reinvent the tests...